ivy/functional/backends/paddle/elementwise.py

```diff
@@ -126,6 +124,10 @@ def equal(
     )
 
 
+@with_unsupported_dtypes(
+    {"2.5.1 and below": ("int8", "int16", "bfloat16", "unsigned", "float16")},
```

Comment:
actually dtypes other then `int8` and `uint8` could be supported by this function as they don't seem to have any manual dtype casting in the implementation

Reply:
As I mentioned in the other discussion, our frame of reference when adding unsupported dtypes for the first time should be the actual framework, but for updating the existing ones in the decorator we should also consider that the dtype might not be actually marked as supported but the tests might've showed that it is and so on. This helps us utilize the work someone else might've put before in adding those dtypes at the start. So if the ivy tests were originally running with `int8` and `uint8` and were passing, then we can consider those dtypes as supported even if that doesn't seem to be marked in the source code. Based on the results of the tests before making these changes, feel free to make the decision of whether or not to add those dtypes. Thanks!

Reply:
understood, I'll keep that in mind.
for this specific example: I tried `int8 `and `uint8` and they are not supported

Reply:
sounds good, thanks!